sdk/src/lib.rs

```diff
@@ -174,10 +283,14 @@ pub use solana_program_memory as program_memory;
 /// ```
 pub use solana_pubkey::pubkey;
 #[cfg(feature = "full")]
-#[deprecated(since = "2.2.0", note = "Use `solana-quic-definitions` crate instead")]
-pub use solana_quic_definitions as quic;
-#[deprecated(since = "2.2.0", note = "Use `solana-rent-debits` crate instead")]
-pub use solana_rent_debits as rent_debits;
+#[deprecated(since = "2.3.0", note = "Use `solana-quic-definitions` crate instead")]
+pub mod quic {
```

Comment:
this needs to die. it's just agave defaults

Reply:
:100: